changelog.txt

```diff
@@ -1,3 +1,6 @@
+0.86.1
+ * Fixed a bug where RealmQuery.findFirst() returned a wrong result if that RealmQuery had been created from RealmResults (#1905).
```

Comment:
`if that RealmQuery` -> `if the RealmQuery had been created from a RealmResults.where()`